### PR TITLE
Warrens work

### PR DESCRIPTION
**Pull request by wyao905**

### CODE REVIEW

File: sections/home-section/landing/index.js [hunk 1]
```diff
@@ -29,6 +48,9 @@ const Landing = () => (
 			</S.ButtonWrapper>
 		</S.ContentWrapper>
 	</S.LandingWrapper>
-)
+}
 
 export default Landing
+
+// /static/gifs/traffic-and-pedestrians-in-front-of-tower-bridge.gif
+// /static/gifs/tourists.gif
```
Comment (naguigui) [inline sections/home-section/landing/index.js:56]: remove me

File: sections/home-section/landing/index.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+const Landing = () => {
+	const [current, setCurrent] = useState(0)
+	const landingGifUrls = [
+		'/static/gifs/financial-business-chart.gif',
+		'/static/gifs/new-york-cityscape.gif'
+	]
+
+	useEffect(() => {
+		const next = (current + 1) % landingGifUrls.length
+		const id = setTimeout(() => setCurrent(next), 5990)
```
Comment (naguigui) [inline sections/home-section/landing/index.js:16]: let's stay away from having magic numbers and move `5990` to a constant at the top of the file

**Review by naguigui — APPROVED** [review]

**Comment by naguigui** [comment]
LGTM! great work on this and happy that you took this on 😄

**Comment by wyao905** [comment]
lol thanks sensei

**Pull request merged**